src/mips/psyqo/cdrom-device.hh

```diff
@@ -321,6 +321,15 @@ class CDRomDevice final : public CDRom {
         S getState() const { return static_cast<S>(m_device->m_state); }
     };
 
+    /**
+     * @brief Checks if the CDROM device is in idle state.
+     *
+     * @details This assumes that all CDRomDeviceStateEnum have a first enum member
+     * with value 0 which corresponds to the "idle" state
```

Comment:
Note that the [concept ](https://github.com/grumpycoders/pcsx-redux/blob/main/src/mips/psyqo/cdrom-device.hh#L47-L55) enforces that already, so the comment is a tiny bit disconnected from the reality.